### PR TITLE
add commit new branch features

### PR DESCRIPTION
**Pull request by GpNt**
add new line

### CODE REVIEW

File: src/Application.java [hunk 1]
```diff
@@ -6,5 +6,7 @@ public static void main(String[] args) {
         System.out.println(4);
         System.out.println(5);
         System.out.println(5);
+        // new
```
Comment (internal-jew) [inline src/Application.java]: This comment is too short

File: src/Application.java [hunk 2]
```diff
@@ -6,5 +6,7 @@ public static void main(String[] args) {
...
+        System.out.println(8);
```
Comment (internal-jew) [inline src/Application.java]: What is this for?

**Comment by GpNt** [comment]
@internal-jew PTAL

**Comment by internal-jew** [comment]
LGTM

**Pull request merged**